content/redirect.js

```diff
@@ -47,7 +48,7 @@ const resolve = (url) => {
       }
     }
   }
-  return redirects.get(url.toLowerCase()) || url;
+  return redirects.get(url.toLowerCase()) || resolveFundamental(url) || url;
```

Comment:
I'm doubtful about this. Particularly from a performance point of view. 

When you run `BUILD_FOLDERSEARCH=web/html yarn build` it builds 238 pages (that match that folder search)
But, before I ran it I put in a 
```javascript
fs.appendFileSync("/tmp/redirectslookup.log", `${url}\n`);`
```
...just to see how often this gets called. And the answer is, a LOT:
```
▶ wc -l /tmp/redirectslookup.log
   30005 /tmp/redirectslookup.log
```

Doing a `Map.prototyoe.get` is fast so I don't worry too much about doing 30k on 15k keys. It's not great but it's fine. Looping over a medium-sized list of regexes 30,000 times is probably a different story.

Reply:
I wouldn't worry to much here. It's only <300 fundamental redirects and probably never that much more for now I guess.

I looked at when I wrote it and the `resolveFundamental` takes _< 20ms_ on a cold start and _~ 300µs_ when jitted (both on my machine).

We can add a cache to that but I'd wait to see it it actually becomes a bottleneck.

Reply:
Generally, I think we can bury the performance topic now. 
As of https://github.com/mdn/yari/pull/1380 we don't look up every single possible URL that passes through KumaScript. Now, we're only looking up things that can't be found on disk.